src/main/java/org/jitsi/jicofo/Main.java

```diff
@@ -124,6 +127,9 @@ public static FocusComponent getFocusXmppComponent()
     public static void main(String[] args)
         throws ParseException
     {
+        Thread.setDefaultUncaughtExceptionHandler((t, e) ->
```

Comment:
What prompted this?

Reply:
that's what the UtilActivator was doing - logging unhandled exceptions

Reply:
but... I'm not 100% sure if that was only it(mentioning that because Boris reacted to my comment). anyway lets try to get rid of it

Reply:
I wasn't able to follow all the usages inside jitsi, but if something was broken we would have seen an exception and there were none in my testing and the PR tests.